src/epcpm/cantoxlsx.py

```diff
@@ -48,31 +52,16 @@ class Fields(epcpm.pm_helper.FieldsInterface):
     description = attr.ib(default=None, type=typing.Union[str, bool])
     access_level = attr.ib(default=None, type=typing.Union[str, bool, int])
     units = attr.ib(default=None, type=typing.Union[str, bool])
-    minimum = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
-    maximum = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
-    c1k_2l_2700_default = attr.ib(
-        default=None, type=typing.Union[str, bool, decimal.Decimal]
-    )
-    c1k_2l_3500_default = attr.ib(
-        default=None, type=typing.Union[str, bool, decimal.Decimal]
-    )
-    c1k_3l1_2700_default = attr.ib(
-        default=None, type=typing.Union[str, bool, decimal.Decimal]
-    )
-    c1k_3l1_3500_default = attr.ib(
-        default=None, type=typing.Union[str, bool, decimal.Decimal]
-    )
-    c1k_3l2_default = attr.ib(
-        default=None, type=typing.Union[str, bool, decimal.Decimal]
-    )
-    pd250_default = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
-    pd500_default = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
-    hy_default = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
     can_path = attr.ib(default=None, type=typing.Union[str, bool])
     parameter_path = attr.ib(default=None, type=typing.Union[str, bool])
     enumerator_list = attr.ib(default=None, type=typing.Union[str, bool])
     parameter_uuid = attr.ib(default=None, type=typing.Union[str, bool])
     epyq_can_parameter_name = attr.ib(default=None, type=typing.Union[str, bool])
+    minimum = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
+    maximum = attr.ib(default=None, type=typing.Union[str, bool, decimal.Decimal])
+    defaults = attr.ib(
```

Comment:
Defaults were moved to the end so that the other columns stay at fixed position regardless of the number of pvmvs files.